web/src/lib/components/map/Map.svelte

```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
+	import { MapHelper, createMap } from "./mapUtils";
+	import { onMount } from "svelte";
+	import {} from "ol/ol.css";
```

Comment:
Is this import necessary? If it is, then you can leave it as `import "ol/ol.css";`